modules/test/poshi-runner/src/main/java/com/liferay/poshi/runner/util/JSONUtil.java

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.poshi.runner.util;
+
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+/**
+ * @author Leslie Wong
+ */
+public class JSONUtil {
```

Comment:
You may want to change the JSONObject getters you are using here to the "opt" getters. The standard getters throw an exception when the desired value is not found. The "opt" getters just return NULL.

```
JSONObject.get() -> JSONObject.opt()
JSONObject.getString() -> JSONObject.optString()
etc...
```